### PR TITLE
Added libuv to the external libs pack for Emotiq

### LINKED ISSUE #3: Add `libuv` to the libraries

### PR DESCRIPTION
**Pull request by echupriyanov**
Closes #3

### CODE REVIEW

**Review by easye — COMMENTED** [review]
(optional) Might be nice to include instructions for a developer to build libraries in the toplevel README.

File: build-libuv.bash [hunk 1]
```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+#
+#  Assuming the proper development tools are in place, make the
+#  libraries needed, and produce a shared object suitable for loading
+#  the code for Pair Based Curves (PBC).
```
Comment (easye) [inline build-libuv.bash]: Small nit: comment refers to PBC curves, but this is the libuv build script.

**Review by easye — APPROVED** [review]

File: build-libuv.bash (hunk 1 above)
Comment (echupriyanov) [reply build-libuv.bash]: Oh, thanks!

**Comment by echupriyanov** [comment]
Created issue #7 to document build process

**Pull request merged**